app/models/katello/deb.rb

```diff
@@ -0,0 +1,89 @@
+module Katello
+  class Deb < Katello::Model
+    include Concerns::PulpDatabaseUnit
+
+    CONTENT_TYPE = Pulp::Deb::CONTENT_TYPE
+
+    has_many :repositories, :through => :repository_debs, :class_name => "Katello::Repository"
+    has_many :repository_debs, :class_name => "Katello::RepositoryDeb", :dependent => :destroy, :inverse_of => :deb
+
+    has_many :content_facets, :through => :content_facet_applicable_debs, :class_name => "Katello::Host::ContentFacet"
+    has_many :content_facet_applicable_debs, :class_name => "Katello::ContentFacetApplicableDeb",
+             :dependent => :destroy, :inverse_of => :deb
+
+    scoped_search :on => :name, :complete_value => true
+    scoped_search :on => :version, :complete_value => true
+    scoped_search :on => :release, :complete_value => true
+    scoped_search :on => :arch, :complete_value => true
+    scoped_search :on => :epoch, :complete_value => true
```

Comment:
doeesn't look like epoch is defined in the migration for the deb table

Reply:
Again, i learned something: The `scoped_search` automatically blends into the search field of the debs-Page. 🎒